callouts/python/extproc/example/validate_token/service_callout_example.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from grpc import ServicerContext
+from envoy.service.ext_proc.v3 import external_processor_pb2 as service_pb2
+from extproc.service import callout_server
+from extproc.service import callout_tools
+
+
+def validation_header_token(request_headers):
```

Comment:
I think the goal of this use case is to showcase how the end user can validate JWT tokens as part of an extension. towards that goal, I suggest we use some python package that takes in JWKS keyset and extracts the JWT token from the "Authorization: bearer <...>" header and validates it. If the token is invalid, you deny the request. If its valid, then you add bunch of headers that provide the decoded JWT fields.

Reply:
@rshriram considering these comments let me know if the #70 covers that, if you agree we can consider this UC part of the jwt PR and close this current one